assets/js/playground_tabs.js

```diff
@@ -9,76 +9,493 @@
+  # to go to the next page!
+) {
+  site {
+    products (first: $pageSize, after:$cursor) {
+      pageInfo {
+        startCursor
+        endCursor
+      }
+      edges {
+        cursor
+        node {
+          entityId 
+          name
+        }
+      }
+    }
+  }
+}`
+
+var categoryTreeQuery = 
+`# Explicitly fetch the first 3 levels of the category tree,
+# and get a few fields on each category
+query CategoryTree3LevelsDeep {
+  site {
+    categoryTree {
+      ...CategoryFields
+      children {
+        ...CategoryFields
+        children {
+          ...CategoryFields
+        }
+      }
+    }
+  }
+}
+
+fragment CategoryFields on CategoryTreeItem {
+  name
+  path
+  entityId
+}`
+
+var lookUpUrlQuery = 
+`# Given a URL path, look it up to see if it matches a Product, Brand, or Category,
+# and fetch details about that object
+query LookUpUrl(
+  $urlPath: String!
+  # use GraphQL Query Variables to provide a path
+) {
+  site {
+    route(path: $urlPath) {
+      node {
+        __typename
+        id
+        # A different response is returned based on which type of object was matched
+        ... on Category {
+          name
+          description
+        }
+        ... on Brand {
+          name
+          defaultImage {
+            url(width: 200)
+          }
+        }
+        ... on Product {
+          name
+          description
+          images {
+            edges {
+              node {
+                url(width: 500, height: 500)
+              }
+            }
+          }
+          brand {
+            name
+          }
+        }
+      }
+    }
+  }
+}`
+
+var singleProductQuery = 
+`# Fetch details about a product by its ID
+query productById(
+  $productId: Int! 
+  # Use GraphQL Query Variables to inject your product ID
+) {
+  site {
+    product(entityId: $productId) {
+      id
+      entityId
+      name
+      plainTextDescription
+      defaultImage {
+        ...ImageFields
+      }
+      images {
+        edges {
+          node {
+            ...ImageFields
+          }
+        }
+      }
+      reviewSummary {
+        summationOfRatings
+        numberOfReviews
+      }
+      prices {
+        price {
+          ...MoneyFields
+        }
+        priceRange {
+          min {
+            ...MoneyFields
+          }
+          max {
+            ...MoneyFields
+          }
+        }
+        salePrice {
+          ...MoneyFields
+        }
+        retailPrice {
+          ...MoneyFields
+        }
+        saved {
+          ...MoneyFields
+        }
+        bulkPricing {
+          minimumQuantity
+          maximumQuantity
+          ... on BulkPricingFixedPriceDiscount {
+            price
+          }
+          ... on BulkPricingPercentageDiscount {
+            percentOff
+          }
+          ... on BulkPricingRelativePriceDiscount {
+            priceAdjustment
+          }
+        }
+      }
+      brand {

```

Comment:
can be removed (?)

Reply:
I think it's relevant here - trying to show a more complete product object. for PDP use cases.